PathOfBuildingCommunity/PathOfBuilding
# Various fixes for Beacon of Madness boots

### Description of the problem being solved:

- Fixes base type variants not matching mods correctly (ar/ev was swapped with es/ev)
- Updates Armour/Evasion base to grant +Maximum Fortification instead of Fortify Effect
- Adds a configuration to enable the granted Embrace Madness skill
- Adds support for all applicable modifiers granted during Glorious Madness effect

### Steps taken to verify a working solution:
- Armour/Evasion Base
  1. Equip item
  2. Enable Embrace Madness skill effect
  3. Verify DPS increases by 20%
  4. Set fortification stacks config to 35 and confirm damage taken from hits decreases appropriately
- Armour/ES Base
  1. Equip item
  2. Enable Embrace Madness skill effect
  3. Verify ignite, chill, freeze, and shock avoidance is set to 100%
  4. Verify the "conflux" configuration appears (not great, but this is how it works everywhere else)
- ES/Evasion Base
  1. Equip item
  2. Equip an elemental-only skill (e.g. frostbolt) and allocate some chance to poison
  3. Enable Embrace Madness skill effect
  4. Verify poison DPS is added to calculation

### Link to a build that showcases this PR:
```
eNrtW21v2zgS_lz_CsLAArtoE1uS32Kku3ASJzUubrN20m4_FYxE29xSoitRTryL_PeboSRbfotpdXvA3WULbGXrmeHMcPhwhmZPf3v0BZmxMOIyeFu2jqtlwgJXejwYvy3f3V4etcq__Vo6vaFq8mF0FnOBb34tvTrVz0SwGRMgVyaKhmOmPmaanC-gaUoDNWEy6NM_ZXglvbfl9zJgZXJPA4-r7JMraBS9pz57Wx5OqCcfyoRGLgu88-WLThTBMw_KxKc8GEr3K1NXoYyneuwZZw996QGu17_5MLjNjcyD_Mhg-KvTG0HnLBwqqkgE_wPl4D8dswvqw_9BGxUxqKo1W8d2tWpXneqJUzspV7YLD6eMeQsh67jmWP9Z6E3IuqMRcxWfsfOQq_MJDdylG43jnZKHo_uxUHwqOAtztjWbu0Tebei3qtVd4FupqLi4GS5tadWOnWqtdlJt2I1m68fISbWQ22nZJ64mZwImpMAoKNsbB1yxgsI3kkcy-A7_8qI7XTyX_j0PCnmYiXZm40NFOyGjH0ZJ9g6ox-Noufp25mGfBvRcRst5a-xCXvMRW0HudL87NMMNgCDMkGjlDQuBxpSZABp7kMBQ5RZhbafF7FseaNs79V2wxyXMeUZfHtiwdwF7gTJTlwc-Y9xMKthYzOK4QNWbzxCs20ZsL3DNlN4FIYtYOMvx8m71qwLpvJqwIAoO2JgZenrNmDu5gi1xQBUzy8ql9dazwUGsUXAQuC04lpnAAcFBwbXgHJ88Bz4wPJ9o6BmQRcDC8Xw44UwYoHUs8yJGMc0LGKZDXuRAv7szGuUXl3Xy_LpJ8WbZwWDnBAGPrZUC9dY_KHITyj-xBBIHynVCX8Y5Hm3WnnU8gRv5fTOZR9yFHVhXlQPmxe4KgbV2MueimuvLGfNhaehiEOrbXM21k1DPBNTGa-4_l5tCHCTRUYq6Xy-kN2YHDXK4xDCeToEiMM9M5S55CIGOeG73PKoboD9A_X9OpyZVEq5G0wGWaOMBrvl4ogLosNZHaZiKmPsyoTIyHyYHNx7iMhbCqPocfuWrSEXvBWuT6qNtOw1q794RbuQDGDPB3jMyWJM5dJ8uq5idq-kyZMFfc2P9K3CjAbqBF4eY3cZjrEtsDnNa0b05PvX8qQyV_vKcCjfSKnvBNFYk0H21zyP3y308GmH7XIYhQt34dy8vu-e3vY_d1Iq8SIST9SWI_XvsApO_sQtPkEOmaY64Ugg6jRh03iMqItDNvXSih0DvrjLCQwOZduUmaNi-3ZBDEE3A2PkZmaw7cRMkNshGQJgVKph5wG7nU4YZEJnGLCVME7Tua42QSSNq5qBueM2mgbl0boRc1DRG6K5gHS5w9zSLWh8WQrLxmuFhIwz5fawMs013JUZ2YIVu5GC-CjVcHWbAtCYyMiItHE2w6V5jlpRQkhhP2wUbMUh3w2Wv6eSWfmVBSpYZMZ7qtRaRCBjzivnR2Rx2pEsM79rxyAoA9qowBt0eG9FY4Pe_x1RwNUcKz317nRyUQvtNool8wOomUYOLO4KgX18nbzpCpRpwjMz2xDM0UB-AdnR9qD9r8_UpKA9cEXvQ0Kb77sJvQe9x7DIZ44npuYwD2EcCLvCoFzdaL3NiTTVqxYFfnYIpKfZKyHsq7KXupV95gJWp1LtFD0bA09rAGwrql8m3xMGeXv86QmAb8_FznynqUUUrPQXuVzAGFW0MPG1RoaO85oWbOAgRwQ0L6_ZsdKJlkzRJtP6guEawgFw8jgYn2nb7jFEXMlKOSJ96ARS1b8jtgzy6lZBS5ExK4Jx1JyIhsZpP3hWfF2vftHT9-5C6LLVr-9QcGOtUZebrerzTJ73kbkPGCE0807JWmunwIf8jAJpjYRwixcJ5ytMY-QDKFnio1Rot5029ZdsnZaJAae53CKuR_sSAShpp4LoebFDee5SuJN_cDa71w6uJUtOoXak8PDwcT6mayBF7hL7y2JV-ZYo_QszYkQ7fEQ5U6cB_Z-PPZ_i3VlTJNJ0mP1NEmf_gk3YfncaH9xJ2D3yHX2YfdEg-cvZAIgblzGSooLjivh_jLyV_Sel_BidOTo6bVbt-Uq9Xm_VmwhvvGFV9Ol0kAGJT0nFSzoES4oJD7EJNl9kUIvAPqP2bVuvYbjitVrPRaqXneqd6FaYzhM9Dlsx2HDHgWRl4nxidykB_nWMqhEJRGnKaZgdSr53EfgBfq3mb3L3v_X7XLW2sjdLa0iA_JxtSJd1sfikln9sEuvX0-YxG2eEpx8bBKqXgBJR-2ERdMzqO4emCCR7y2C99TGxuk71G7EfqDZokO_Rz-FTlukBSJgNsIWmV_k5j2rafDIddSDhPpgMvRKynvVFIt6o2saulNN3b5OpI_ynp9Buwb23SrJag6BXc5fjeyQ_xNzQzY9auHtefXv_cOrLsX34iShLszgkFysYumiS1Axa1UT4G-2QXvfF2Bc4uBXrM3QquQAqCod0jFlnjO6LpreRUf8KdI2SQdB7JjnKIbify_r-26jikTx-5H_vkEqoCPuIu1TXMwwQSlVDNd6Dlfk6Ax0Mu42ixWnKqbBjS1aU_avQYFeRCxkDXJKl6DlJnP32WMZnQGSO6nIcgvCHnoEBgODA4wwlMOH44l8FIxI8Hqu9pUkNLoVbH2IC5Wcl-kCrnCRLYh76PzMHif-Fevl888YyS2krMuo9TAXvCGx087SdufiFsOkhQasJ4SLCUh72J6BORNLJ5YzowfhpvDnHhsFORpHcy8Qr3DejFB5iXRGcnHnLXEwZNm-xnEPZeRHYUoQusLWRtZ7DvJOtVTjGi7LwIsESjlO9x_tvJ2_4uKj6c71_I-4W8X8j7_4y8nQRWL0reO2jIrJj-3-Zv56X4fuHvF_5-4e8fwN_4qrbK2YPOoFvK7hAQ6x3Y6DKg8AnzpZokn85DOgJX2gQPcko3IRvxxza5liDRy5J_9RpCSsD1Q8CdwOu4bgyrbD6gmIlr0h3PW5dslIbxaIv-5PaAXoXOKgR_O0P5oQr51_SiwA7I8g5qbTsDDo7gz07-s0qNPDckJqXMYDXt_LtF-NM8A08jkjCG3dp4a9k79dorrzJHSOIsSbwtvbabmv-Sk9oN1NLt0mvLshCZTQtJ5mU1sYZCZr_YJmd0BI_hVHLKnt1yyYHSo-YMkd1OSQ_9njnwSxdHeuS5OerwgU5J534eReiMniRSXw5UzRbNFtF1MdtMbMPPTUcLGrTNlz1GnTGhDhS5FDT6Shwjk-xtJjnFwmsWBPsfEqsVM7JRbLRG8WhaeyZ4Wz7tmYF3DLZadehAWzKpUWScPekHBDRjURHFTvFVZBUZr1YsF6ximVfbPz0mgT1U7bb0Kui4Ee_UC9hTgAOdfcN4c5I0PAdO3_YoF8qvehEhoyVpMhGF_KwXDqtdYN4bxVaSUyx9ncOzrFac9gvktFU4-I5JQtRMQAXM_o7ix3Ami89Co0BWFp-G70gXU0pGWcNVYw4tELkBdth2EYqxC4e3IF2Yz2cRdxyDOFnFp6FWOFr1It6YrJeNLs84yfeY1PFjsWVvS7pSaBn19Rd9pUTfSJbBiI837hdjk8nxAE5f-ITol8m9lILRIL2xsnklGS8wp-dgyzvMeKNuE8oC5s970ZnEy04ZdMjDONoCduNISb8vvRx2C8xftOYrB4jLy9FOfZslIo70P_LIX7zud_7o9e_62dXENEKnlfV_f_1vd77SoQ==
```

### Before screenshot:

### After screenshot:
![image](https://user-images.githubusercontent.com/45536/140656868-bb4a265c-c02b-4a70-8841-9ca9144c97cc.png)

![image](https://user-images.githubusercontent.com/45536/140657368-1954eca9-fb70-49ad-b4df-1a26aee0a3d6.png)

![image](https://user-images.githubusercontent.com/45536/140656907-5788eec6-e560-4e48-9d33-fd592c44b536.png)
